Assets/Scripts/RedRunner/Characters/RedCharacter.cs

```diff
@@ -326,6 +326,7 @@ private void LeftEvent()
 				{
 					StopWallSlide();
 					Jump();
+					ApplyHorizontalBoost(m_DoubleJumpStrength/2.0f);
```

Comment:
why are left and right wall jump boosts not equal?

Reply:
whoops missed that